gapless5.js

```diff
@@ -920,6 +929,16 @@ function Gapless5(options = {}, deprecated = {}) { // eslint-disable-line no-unu
     }
   };
 
+  this.setPan = (pan) => {
+    this.pannerNode.pan.value = pan;
+    if (this.currentSource()) {
+      this.currentSource().setPan(pan);
+    }
+    if (this.hasGUI) {
+      getElement('pan').value = scrubSize * pan;
+    }
```

Comment:
When`this.hasGUI` is true, I'm running into the following error:
```
Uncaught TypeError: Cannot set properties of null (setting 'value')
```

Looks like you copy+pasted `setVolume` when you added `setPan`, but you should delete this code block because there is no UI for panning like there is for volume:

```suggestion
```